webvtt.html

```diff
@@ -542,6 +542,17 @@
    </section>
 
    <section>
+    <h3>Unicode normalization</h3>
+
+    <p>This specification does not normalize Unicode text as part of its processing
+    requirements.</p>
+
+    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
```

Comment:
Normalization isn't optional, it's not part of the processing requirements and thus forbidden. In other words, there's no interop issue here if people implement what the spec says.